rust-escrow/ft-factory/src/lib.rs

```diff
@@ -53,6 +53,7 @@ impl FtFactory {
             .parse()
             .unwrap();
         let predecessor_account_id = env::predecessor_account_id();
+        let symbol = format!("NHA{}", self.ft_index.len() + 1);
```

Comment:
Nice.
At least we have a symbol now. 

Should we let the user set the symbol as a future feature?

Reply:
It could be a possibility. I'm going to create that option as well and if no Symbol is provided so we can use the correlative